Source/IntegrationTests/TestFiles/LitTests/LitTest/cli/inputFormatDeleteSources.dfy

```diff
@@ -0,0 +1,45 @@
+// RUN: %tobinary %s --delete-sources > %t.deleteSources.dbin
+// RUN: ! %verify --input-format Binary --allow-warnings --stdin < %t.deleteSources.dbin > %t
+// RUN: %diff "%s.expect" "%t"
+
+method Foo() {
+  assert false; // error
+}
+
+method Bar() {
+  assert false; // error
+}
+
+function F(): int {
+  3
+} by method {
+  assert false; // error
+  return 1 + 2;
+}
+
+// opaque predicate P() { true }
```

Comment:
Worth a comment explaining why these are here (I think it's "these are good to test too but they aren't working correctly yet"?)

Or would it be possible to uncomment them anyway? You made a bunch of nullability declaration changes and updated the schema, but those new cases for the binary format aren't being tested otherwise AFAICT.